azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -705,7 +705,8 @@ private static Props loadConfigurationFromAzkabanHome() {
 
   private void startWebMetrics() throws Exception {
 
-    final MetricRegistry registry = MetricsManager.INSTANCE.getRegistry();
+    final MetricRegistry registry = SERVICE_PROVIDER.getInstance(MetricsManager.class)
```

Comment:
This is being refactored to rely on MetricsManager dependency which is handled in constructor. The MetricsManager class is used explicitly on line 736 to take advantage of the startReporting method. Do you think it'd be better to depend on both MetricsManager and MetricsRegistry?

Also, MetricsRegistry is binded to a singleton locally, I'm resolving some consequences of that switch then I'll push a new version - thanks for the feedback! :+1: